test/.gitattributes

```diff
@@ -0,0 +1,4 @@
+# Remove the text attribute from reference files, so that git doesn't convert
+# line separators on Windows machines. It causes the index files to become out
+# of sync with the fasta files.
+*.fa* -text
```

Comment:
I think the problem here is that git clone itself changes these files.  We can't really regenerate the .fai files on a git clone, unless we have the Makefile *always* regenerate .fai from .fai.  Even this is problematic as there is no htslib command-line interface to fai_build: our only calls to it are over in ```samtools faidx``` instead.